### DISCUSSION TITLE
add reference temperature to climate

### DISCUSSION THREAD

**Discussion by pbranly** [Blackymas/NSPanel_HA_Blueprint#795] [Ideas]
Hi,
I am a new user of this fantastic system and discover everyday.
I found how adding several climates (TRTVs on my side)
It displays the current temperature but not the objective temperature while it seems there are plenty of place. I can access to the settings by clicking on the icon but I find interesting to have both the current and objective temperature on each of every climate.
I join a photo ( dont care of red errors ..... those TRVs are currently disconnected)
Phil
![IMG_4537](https://user-images.githubusercontent.com/22018509/236669834-02856a5d-cc30-4c31-ade5-22aa303c9c14.JPG)

**Comment by edwardtfn** [comment]
- #1726
